.github/workflows/COMMENT_DEVELOP_FIX.yml

```diff
@@ -1,6 +1,8 @@
 name: COMMENT_DEVELOP_FIX
 on:
   pull_request:
+    types:
```

Comment:
Do we want this? 

In this case we'd miss out on (future) cleaned-up commit history hints.

Reply:
Did you test this before locally or so Maciej? I tried multiple cases locally and the behavior (at least for me) was not always predictable

Reply:
I haven't, but we can use this branch for tests.

Reply:
Let me have a test PR.

Reply:
It works on opened: https://github.com/camunda/camunda-modeler/pull/2949

Reply:
But doesn't trigger a new comment on additional commits.